notebooks/concurrent_tasks.ipynb

```diff
@@ -0,0 +1,1226 @@
+{
```

Comment:
<p>Maybe not say "workaround"? Just say "we show how you can"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
done